cypress/integration/seededFlows/readingListFlows/readingListArchive.spec.js

```diff
@@ -81,7 +81,7 @@ describe('Reading List Archive', () => {
     cy.get('@main')
       .contains('Test Article 1')
       .parents('article')
-      .findByLabelText(/^Archive item$/i)
+      .find('.spec__archive-button')
```

Comment:
Refactor (blocking): Use `.findByLabelText` or another cypress-testing library method liek `findByRole`.

Reply:
I definitely need a help with that. Previously it made sense to have "Archive item" because [we had that `aria-label`](https://github.com/forem/forem/pull/15843/files#diff-5cbe72b80ca710821e016d51942b33b5bae44c96cfce59508776b78b13afd4e0L15) on the element. But after talking to @aitchiss about it I figured it didn't really make sense because that button label is dynamic (depending on situation it can either be "Archive" or "Unarchive" yet the `aria-label` was static and it always said "Archive item" which didn't really make sense for Unarchive button... Suzanne suggested that this aria-label will be more harmful than useful in such a scenario.

I suck at cypress so I'd definitely appreciate the help.

Reply:
@ludwiczakpawel, I've updated the tests to use `findByRole`. See [`602f9f6` (#15843)](https://github.com/forem/forem/pull/15843/commits/602f9f6d090d285c78410e588f56044afe379401)

Reply:
thank you!